attestation-agent/kbs_protocol/src/lib.rs

```diff
@@ -93,10 +93,10 @@ impl KbsProtocolWrapper {
         &mut self.http_client
     }
 
-    async fn attestation(&mut self, kbs_host_url: String) -> Result<String> {
+    async fn attestation(&mut self, kbs_root_url: String) -> Result<String> {
```

Comment:
maybe use `kbs_url` directly which contains the `scheme://domain-name[:port][/user]`

Reply:
In many domain name allocation scenario, the root domain name assigned to KBS may have a more complex form. I believe we should leave enough flexibility for the form of the root domain name. Therefore, for now, I suggest not defining and resolving the format and content of the root domain name.

Reply:
> maybe use `kbs_url` directly which contains the `scheme://domain-name[:port][/user]`

If we use `kbs_url` here, it may be confused with the whole resource url.